airflow/providers/cncf/kubernetes/decorators/kubernetes.py

```diff
@@ -100,14 +100,8 @@ def _generate_cmds(self) -> list[str]:
         return [
             "bash",
             "-cx",
-            " && ".join(
-                [
-                    write_local_script_file_cmd,
-                    write_local_input_file_cmd,
-                    make_xcom_dir_cmd,
-                    exec_python_cmd,
-                ]
-            ),
+            f"{write_local_script_file_cmd} && {write_local_input_file_cmd} && {make_xcom_dir_cmd} &&"
+            f" {exec_python_cmd}",
```

Comment:
Should we try something like this to make it more readable?

```suggestion
            (
                f"{write_local_script_file_cmd} && "
                f"{write_local_input_file_cmd} && "
                f"{make_xcom_dir_cmd} && "
                f"{exec_python_cmd}"
            )
```

Reply:
If Black is OK with this, yes good idea

Reply:
just did an experiment locally. seems to work fine